+hw/calibrate.m

```diff
@@ -37,12 +37,16 @@
 
 % Check inputs
 narginchk(5,15)
-assert(~mod(length(varargin),2), 'Rigbox:hw:calibrate:partialPVpair', ...
-  'Incorrect number of Name-Value pairs')
+if ~all(cellfun(@ischar, (varargin(1:2:end)))) || mod(length(varargin),2)
+  error('Rigbox:hw:calibrate:nameValueMismatch', ...
```

Comment:
I chose the id `partialPVpair` because that's exactly the one MATLAB used in a builtin function.  Whether they themselves have standardized all their IDs moot (they probably don't).

Reply:
Oh I see. I use `nameValueMismatch` in my own code. That's fine, feel free to make any changes you see fit before merging